dnf.spec

```diff
@@ -43,6 +43,7 @@ Requires:       python3-%{name} = %{version}-%{release}
 %else
 Requires:       python2-%{name} = %{version}-%{release}
 %endif
+Recommends:     (python%{?with_python3:3}-dbus if NetworkManager)
```

Comment:
This is not allowed due to [`mash`](https://pagure.io/mash) not being ported to DNF. Yum can read `Requires`, `Recommends`, and `Suggests`, so rich deps cannot be used there unless `mash` is ported to DNF.